blocks/header/headerutil.js

```diff
@@ -0,0 +1,309 @@
+const ENIITY_TYPE = {
+  ALL: 'all',
+  EQUITY: '',
+  MUTUAL_FUND: 'MF',
+  CURRENCY: 'currency',
+  COMMODITY: 'commodity',
+  KNOWLEDGE_CENTER: 'ilearn',
+  IDX: 'IDX',
+  DERIVATIVE: 'derivative',
+  BONDS: 'bonds',
+};
+
+const SITE_ROOT = 'https://www.icicidirect.com';
```

Comment:
Can we resuse constant from block-utils here ?  or if ist not there we can move this constant in block-utils

Reply:
done moved it into block-utils.